tests/bfd/test_bfd.py

```diff
@@ -105,72 +142,104 @@ def stop_ptf_bfd(ptfhost):
+        if idx%50 == 0:
+            ptfhost.shell(cmd_buffer)
+            cmd_buffer = ""
+    if cmd_buffer != "":
+        ptfhost.shell(cmd_buffer)
 
 def del_ipaddr(ptfhost, neighbor_addrs, prefix_len, neighbor_interfaces, ipv6=False):
+    cmd_buffer = ""
     for idx in range(len(neighbor_addrs)):
         if ipv6:
-            ptfhost.shell("ip -6 addr del {}/{} dev eth{}".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx]), module_ignore_errors=True)
+            cmd_buffer += "ip -6 addr del {}/{} dev eth{} ;".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx])
         else:
-            ptfhost.shell("ip addr del {}/{} dev eth{}".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx]), module_ignore_errors=True)
-
+            cmd_buffer += "ip addr del {}/{} dev eth{} ;".format(neighbor_addrs[idx], prefix_len, neighbor_interfaces[idx])
+        if idx%50 == 0:
+            ptfhost.shell(cmd_buffer)
+            cmd_buffer = ""
+    if cmd_buffer != "":
+        ptfhost.shell(cmd_buffer,  module_ignore_errors=True) 
 
 def check_ptf_bfd_status(ptfhost, neighbor_addr, local_addr, expected_state):
     bfd_state = ptfhost.shell("bfdd-control status local {} remote {}".format(neighbor_addr, local_addr))["stdout"].split("\n")
     for line in bfd_state:
         field = line.split('=')[0].strip()
         if field == "state":
-            assert line.split('=')[1].strip() == expected_state
+            assert expected_state in line.split('=')[1].strip()
 
 
 def check_dut_bfd_status(duthost, neighbor_addr, expected_state):
     bfd_state = duthost.shell("sonic-db-cli STATE_DB HGET 'BFD_SESSION_TABLE|default|default|{}' 'state'".format(neighbor_addr), module_ignore_errors=False)['stdout_lines']
-    assert bfd_state[0] == expected_state
-
+    if not SkipStateCheckAssert:
+        assert bfd_state[0] == expected_state
+    else:
+        print ("Expected State ", expected_state, "actual state", bfd_state[0])
 
-def create_bfd_sessions(ptfhost, duthost, local_addrs, neighbor_addrs):
+def create_bfd_sessions(ptfhost, duthost, local_addrs, neighbor_addrs, dut_init_first, scale_test = False):
     # Create a tempfile for BFD sessions
     bfd_file_dir = duthost.shell('mktemp')['stdout']
     bfd_config = []
-
+    dut_buffer = ""
+    ptf_buffer = ""
+    if scale_test:
+        # Force the PTF initialization to be first if runnign scale test. Doing so that we can send bathces fo 50 commadns to ptf
+        # and keep the code readable.
+        assert( dut_init_first == False )
+    
     for idx, neighbor_addr in enumerate(neighbor_addrs):
-        duthost.shell("sonic-db-cli APPL_DB hmset 'BFD_SESSION_TABLE:default:default:{}' local_addr {}".format(neighbor_addr, local_addrs[idx]))
+        dut_buffer += "sonic-db-cli APPL_DB hmset 'BFD_SESSION_TABLE:default:default:{}' local_addr {} ;".format(neighbor_addr, local_addrs[idx])
```

Comment:
Correct me if I'm wrong, but this cmd `sonic-db-cli APPL_DB hmset` is useless. I've tried running it manually and it doesn't work. Moreover, we're creating BFD sessions via applying `swssconfig` below. So we can get rid of it.